meson.build

```diff
@@ -1,5 +1,5 @@
 project('curtail',  
-          version: '1.3.1',
+          version: '1.4.0',
     meson_version: '>= 0.50.0',
   default_options: [ 'warning_level=2',
                    ],
```

Comment:
Prefer using 
```
gnome = import('gnome')

gnome.post_install(
  gtk_update_icon_cache: true,
  glib_compile_schemas: true,
  update_desktop_database: true,
)
```

instead of custom postinstall.py scripts. This requires meson 0.59.